### ISSUE TITLE
Improve README, include animated gif showing compareWith in action

### ISSUE THREAD

**Issue by riccardoporreca** [miraisolutions/compareWith#9]
Use a dummy example of a project under Git with existing merge conflicts.

To be done after #8, which might change the list of addins. 

### Setup an RStudio project for a checkout with conflicts and only compareWith Addins

1. Follow the instructions in [miscellaneous/foobar/README.md](https://github.com/miraisolutions/miscellaneous/blob/master/foobar/README.md) to create a local checkout with merge conflict.
2. Create an RStudio project on the foobar directory.
3. Create an `.Rprofile` defining a dedicated .libPath() as follows
``` r
# Dedicated library path where only compareWith should be installed to have a
# clean Addins menu
lib <- paste(Sys.getenv("R_LIBS_USER"), "compareWith", sep = "-")
dir.create(lib, showWarnings = FALSE)
message("Set library path to ", sQuote(lib))
.libPaths(lib)
```
4. Added as `foobar/.Rprofile` and `foobar/foobar.Rproj` to the `.gitignore` in the parent directory)
5. Restart the R session and install compareWith
``` r
install.packages("remotes")
remotes::install_github("miraisolutions/compareWith")
```
6. Restart the R session an check the _Addins_.
7. Open `foo.R` and `bar.R`, with `bar.R` as active document, showing the conflicts. 

### Steps for the animation

0. Rstudio with diff / conflicts in the Git pane.
1. Addins menu with mouse over "Compare with repo (project)".
2. Meld window showing the list of blue / red files, and mouse over a red conflicted file.
3. Three-way comparison of the conflicted file.
4. Three-way comparison with resolved conflict and closing popup saying "Mark conflicts as resolved". (Optional, might be hard if we want a _continuous_ usage)

#### Detailed process

Required tools (Linux): `peek`, `gifsicle`, `convert`.

**Record the animation** (the main challenge is to have straight mouse paths, which require practicing to get used to where things are located).
0. Copy `na.rm = TRUE` to the clipboard.
1. Open RStudio and set the default light color scheme, set the panes geometry as in the existing GIF`man/figures/compareWith-RStudioAddins.gif`, select the conflicted "bar.R" in the Git pane. 
2. Run the "Compare with repo (project)" and set the Meld window size and position as in the existing GIF, leave it open, we will not re-launch it during recording since the position will be lost.
3. Launch Peek, and set the "delay before start" preference to 10 s. Then resize the capture window to fit the RStudio window excluding the title. 
4. Start recording, and in the 10s go to Meld first (important for switching back to Meld later) and then RStudio, position the mouse on the "bar.R" in the Git Pane and wait for the count-down to finish.
5. Go to and click the _Addins_ menu, mousover _Compare with repo (project)_ and wait for the tooltip to appear, then quickly press _Esc_ and _Alt+Tab_. This will bring you back to the previously-open Meld, giving the impression you just launched it.
6. Double-click to bar.R in Meld to show the 3-way diff/merge with the conflicts
7. Click on the right-to-middle arrows for each of the 2 conflicted chunks
8. In the middle file, click and _Ctrl+V_ between the parentheses of the `foo_sum()` and `foo_prod()` calls to enter `na.rm = TRUE`.
9. Press _Ctrl+S_ and click on the x in the foo.R tab to close, then click "Mark as Resoved" in the prompt.
10. Press _Ctrl+Altr+R_ to stop recording, save the file as compareWith-raw.gif

**Post-process the animation** (remove unwanted frames at the beginning / end, add progressive blurring)
1. Open the console and `cd` to the location of the saved file.
2. Create directory `mkdir frames` and `cd frames`
3. Extract all frames `gifsicle -U -E ../compareWith-raw.gif`
4. Check initial and final frames if any should be excluded (e.g. due to the Peek countdown being displayed)
5. Given the last frame, e.g. `compareWith-raw.gif.126`, create blurred copies via 
```bash
for i in {1..8}; do convert compareWith-raw.gif.126 -blur 0x$i zzz-blur.$i; done
```
6. Recreate the full GIF `gifsicle -O3 * -o ../compareWith-RStudioAddins.gif`

**Comment by nfarabullini** [comment]
@riccardoporreca I created a new branch for this issue and pushed a couple of edits on the README

**Comment by riccardoporreca** [comment]
Closed by #13.
